config/routes.rb

```diff
@@ -1,7 +1,12 @@
 Rails.application.routes.draw do
   resources :acciones
-  resources :accion_en_snapshots
-  resources :snapshots
+  resources :snapshots do
+    resources :accion_en_snapshots
+  end
+  get 'importar_snapshot/establecer_origen' => 'snapshot_importer#establecer_origen', as: 'establecer_origen'
+  post 'importar_snapshot/pide_lista_snapshots' => 'snapshot_importer#pide_lista_snapshots', as: 'pide_lista_snapshots'
```

Comment:
Line is too long. [119/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.